src/sentry/db/models/utils.py

```diff
@@ -54,7 +54,7 @@ def _resolve(instance, node):
 resolve_expression_node = resolve_combined_expression
 
 
-def slugify_instance(inst, label, reserved=(), max_length=30, field_name="slug", *args, **kwargs):
+def slugify_instance(inst, label, reserved=(), max_length=50, field_name="slug", *args, **kwargs):
```

Comment:
Any reason to not change the call sites instead of the default? Changing the call sites would prevent any unintended side effects.

Reply:
@markstory I can add it, but IMHO it's better to carefully check just once instead of always keeping in mind to provide value for `max_length` for any function usage in future code (to introduce same unexpected truncation). 
Might be even better approach would be to set default to `None` and assign `max_length` dynamically from the field if no value given.